### PR TITLE
[KHO-214] Feature/basic metrics

### PR DESCRIPTION
**Pull request by nandini-ghosh**
# Checklist

- [x] I have performed a self-review of my code
- [x] I have reached out to another developer to review my code
- [x] I acknowledge my responsibility to reinitialize the deployed DB if there are schema changes.
- [x] New and existing unit tests pass locally with my changes
- [x] All Lint and CI checks are passing

UI Components for basic text-based metrics, styling fixes, integrating metrics endpoints for Dashboard page

NOTE: Minor DB change to test TA:Student ratio in dashboard

### CODE REVIEW

**Review by kennybc — CHANGES_REQUESTED** [review]

File: frontend/src/pages/Dashboard/index.tsx [hunk 1]
```diff
@@ -88,54 +128,62 @@ const Dashboard: React.FC = () => {
...
+                          "Professor",
+                          classroomUsersList.filter(
+                            (user) => user.classroom_role === "PROFESSOR"
+                          )
+                        )
+                      }
+                    />
+                  </div>
+
+                  <SimpleMetric metricTitle="Created on" metricValue={formatDate(selectedClassroom.created_at ?? null)}></SimpleMetric>
```
Comment (kennybc) [inline frontend/src/pages/Dashboard/index.tsx:180]: I see "Dec 31, 1" from this which definitely is not right

**Review by kennybc — CHANGES_REQUESTED** [review]

File: frontend/src/pages/Dashboard/index.tsx (hunk 1 above)
Comment (nandini-ghosh) [reply frontend/src/pages/Dashboard/index.tsx:180]: Huh that's odd, can you attach a ss? I'm using a function you'd defined in utils and it works correctly everywhere else

Comment (nandini-ghosh) [reply frontend/src/pages/Dashboard/index.tsx:180]: done

**Review by kennybc — APPROVED** [review]

**Review by kennybc — APPROVED** [review]

**Review by alexangione419 — APPROVED** [review]
epic

**Pull request merged**